tools/db/repair-brig-clients-table/repair-brig-clients-table.cabal

```diff
@@ -0,0 +1,83 @@
+cabal-version: 1.12
+name:          repair-brig-clients-table
+version:       1.0.0
+synopsis:
+  Removes and reports entries from brig.clients that have been accidentally upserted.
+
+category:      Network
+author:        Wire Swiss GmbH
+maintainer:    Wire Swiss GmbH <backend@wire.com>
+copyright:     (c) 2018 Wire Swiss GmbH
+license:       AGPL-3
+build-type:    Simple
+
+executable repair-brig-clients-table
+  main-is:            Main.hs
+  other-modules:
+    Options
+    Paths_repair_brig_clients_table
+    Work
+
+  hs-source-dirs:     src
+  default-extensions:
```

Comment:
I think there's some overlapping with what `GHC2021` brings... However, this is just a one-shot cleanup tool. :man_shrugging:

Reply:
yeah, i wouldn't want to spend time on this now...